src/test/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticatorTest.java

```diff
@@ -111,6 +111,65 @@ public void testInvalid() throws Exception {
         Assert.assertNull(credentials);
     }
 
+    @Test
+    public void testParsePrevGeneratedJwt() {
```

Comment:
I don't normally suggestion including commented out code, but I think this is a very good case - could you include the code used to generate the previous token and a link to where you injected it?

Reply:
I just added